### PR TITLE
Sector react typeahead

### PR DESCRIPTION
**Pull request by rachaelcodes**
## Description of change

Adds a typeahead to the investment project react view to allow searching by Sector.

_This will need further refactoring when other typeaheads using the metadata are added - I didn't want to make it too generic before it was clear what would be needed._

## Test instructions

Run locally; search for a project sector (currently hardcoded to the values in the fixture database - will make a new ticket to update the BE fixtures to use the updated ids before adding the new sector id list). The number of projects should change, and a chip appear with the filter name. If you copy and paste the URL to a new browser tab, the filters should still be visible and working.

## Screenshots
Screenshot 2020-11-19 at 11 06 17

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to ~master~ feature-merge branch?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

**Review by ian-leggett — APPROVED** [review]
I can start to see some commonalities appearing so in the next PR we can look at making some abstractions. This works really well 👍

File: src/client/components/FilteredCollectionList/FilteredCollectionHeader.jsx [hunk 1]
```diff
@@ -68,6 +70,14 @@ function FilteredCollectionHeader({
           <AdviserFilterChips selectedAdvisers={selectedAdvisers} />
         </CollectionHeaderRow>
       )}
+      {hasOptions && (
+        <CollectionHeaderRow>
+          <OptionFilterChips
+            selectedOptions={selectedSectors}
+            keyName="sector"
```
Comment (tomkingston) [inline src/client/components/FilteredCollectionList/FilteredCollectionHeader.jsx:77]: Hi Rachael - just trying to understand how this work - is the keyName picked up by the OptionFilterChips component, or can I ignore it?

Comment (tomkingston) [reply src/client/components/FilteredCollectionList/FilteredCollectionHeader.jsx:77]: Actually - I can probably just use the Chip component, so don't worry!

Comment (rachaelcodes) [reply src/client/components/FilteredCollectionList/FilteredCollectionHeader.jsx:77]: I think we're planning on refactoring this into a general one in another ticket

Comment (rachaelcodes) [reply src/client/components/FilteredCollectionList/FilteredCollectionHeader.jsx:77]: but, good shout, that was something I'd taken out and not removed the reference to here - will update

**Comment by rachaelcodes** [comment]
Changed branch name - am reopening this with different changes

**Pull request closed without merging**